tool/src/test/java/org/wildfly/security/tool/SecretKeyCommandTest.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2021 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.security.tool;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import javax.crypto.SecretKey;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+import org.wildfly.security.credential.SecretKeyCredential;
+import org.wildfly.security.credential.store.CredentialStore;
+import org.wildfly.security.encryption.CipherUtil;
+import org.wildfly.security.encryption.SecretKeyUtil;
+
+/**
+ * Test case to cover {@code SecretKey} management using the credential-store command.
+ *
+ * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
+ */
+@RunWith(Parameterized.class)
+public class SecretKeyCommandTest extends AbstractCommandTest {
```

Comment:
I suggest to add also tests for related actions that already exist: `exists`, `remove`, `aliases`, using the parameter `entry-type`.

Reply:
BTW, how can I (as a user) figure out which values can be used for `entry-type`?

Reply:
It seems `aliases` doesn't work as expected and there is a missing space in the second output:

```
$ ./bin/elytron-tool.sh credential-store --location a --password a --aliases --entry-type PasswordCredential
Credential store contains following aliases: a b 
$ ./bin/elytron-tool.sh credential-store --location a --password a --remove b --entry-type PasswordCredential
Alias "b" of type "PasswordCredential"does not exist
```

Reply:
https://issues.redhat.com/browse/ELY-2093